### PR TITLE
Use less specific types in validators args

### PR DESCRIPTION
**Pull request by kalekseev**
Make it possible to use Tuple[str]

### CODE REVIEW

File: django-stubs/core/validators.pyi [hunk 1]
```diff
@@ -38,7 +38,7 @@ class URLValidator(RegexValidator):
     tld_re: Any = ...
     host_re: Any = ...
     schemes: Any = ...
-    def __init__(self, schemes: Optional[List[str]] = ..., **kwargs: Any) -> None: ...
+    def __init__(self, schemes: Optional[Iterable[str]] = ..., **kwargs: Any) -> None: ...
```
Comment (mkurnikov) [inline django-stubs/core/validators.pyi]: From the implementation, it looks like it should be a `Container`, `Iterable` is not enough. Could you make it a `Collection`, it implements both interfaces?

File: django-stubs/core/validators.pyi [hunk 2]
```diff
@@ -52,7 +52,7 @@ class EmailValidator:
     literal_regex: Any = ...
     domain_whitelist: Any = ...
     def __init__(
-        self, message: Optional[str] = ..., code: Optional[str] = ..., whitelist: Optional[List[str]] = ...
+        self, message: Optional[str] = ..., code: Optional[str] = ..., whitelist: Optional[Iterable[str]] = ...
```
Comment (mkurnikov) [inline django-stubs/core/validators.pyi]: Here too.

File: django-stubs/core/validators.pyi [hunk 3]
```diff
@@ -119,7 +119,7 @@ class FileExtensionValidator:
     code: str = ...
     allowed_extensions: List[str] = ...
     def __init__(
-        self, allowed_extensions: Optional[List[str]] = ..., message: Optional[str] = ..., code: Optional[str] = ...
+        self, allowed_extensions: Optional[Iterable[str]] = ..., message: Optional[str] = ..., code: Optional[str] = ...
```
Comment (mkurnikov) [inline django-stubs/core/validators.pyi]: All of them actually.

**Comment by kalekseev** [comment]
@mkurnikov replaced Iterable with Collection.

Also what do you think about adding pyproject.toml file with black configuration (line length = 120) otherwise editors reformats with default line length?
I can also setup pre-commit with black, isort and flake8.

**Comment by mkurnikov** [comment]
Sure, I would gladly accept a PR with those changes. Let's start with the `pyproject.toml`, and add pre-commit hooks later.

**Pull request merged**